src/store/plugins/names.js

```diff
@@ -116,19 +112,19 @@ export default (store) => {
         commit,
       }) {
         commit('setAreNamesFetching', true);
-        const fetchPendingNameClaimTransactions = (address) => fetchPendingTransactions(address)
-          .then((transactions) => transactions
-            .filter(({ tx: { type } }) => type === 'NameClaimTx')
-            .map(({ tx, ...otherTx }) => ({
-              ...otherTx,
-              ...tx,
-              owner: tx.accountId,
-            })));
+        const adapter = ProtocolAdapterFactory.getAdapter(PROTOCOL_AETERNITY);
 
         const middleware = await getMiddleware();
         const names = await Promise.all(
           aeAccounts.value.map(({ address }) => Promise.all([
-            fetchPendingNameClaimTransactions(address),
+            adapter.fetchPendingTransactions(address)
+              .then((transactions) => transactions
+                .filter(({ tx: { type } }) => type === 'NameClaimTx')
```

Comment:
In my opinion mixing `Promise.all` with `.then` and filtering in one block of code is rather bad practice. It makes the code hard to read.
I suggest to first fetch the data and make the operations on existing constant.